### ISSUE TITLE
[BUG] "In all sub-folders" when searching does not go through all sub-folders

### ISSUE THREAD

**Issue by TeKett** [notepad-plus-plus/notepad-plus-plus#15084]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description of the Issue

It does not go through sub-sub-folders, or sub-sub-sub-folders, etc, they are also sub-folders that should be searched

### Steps To Reproduce

Do a search for multiple layers of sub-folders

### Current Behavior

Only searches through the current and one layer of sub-folders

### Expected Behavior

Searching through any and all layers of sub-folders, even a million deep.

### Debug Information

```shell
-
```

**Comment by alankilborn** [comment]
>It does not go through sub-sub-folders, or sub-sub-sub-folders, etc

Yes it does, in v8.6.5; not sure what version you are using, that was supposed to be supplied by you following directions and supplying the _Debug Information_.

**Comment by TeKett** [comment]
ok, i figured out what the issue is, for whatever reason i cant search for spaces when i search in files, not even with extended, or regex. Which is strange, since it was working and only started to not work when i wanted to replace in sub-sub-folders. Now i literally cant search for spaces no matter what i do unless i open the file and replace in open files.

**Comment by alankilborn** [comment]
@TeKett I think you need to show exactly what you are attempting.  Can you paste a screenshot of the _Find_ window, and otherwise more exactly describe what you're doing?  Note: for pasting a screenshot, since your data apparently involves spaces, can you make sure that the _Find what_ data is "selected" before you capture the screenshot?
